spec/views/components/_attachment_meta.html.erb_spec.rb

```diff
@@ -0,0 +1,58 @@
+RSpec.describe "components/_attachment_meta.html.erb", type: :view do
+  it "fails to render when no attachment is given" do
```

Comment:
Minor: this seems like an unusual test, compared to other components I've seen. We just shouldn't be using the component in this way, so it's OK for it's behaviour to be undefined in the tests.

Reply:
This is a [pretty common test in govuk_publishing_components](https://github.com/alphagov/govuk_publishing_components/blob/master/spec/components/contents_list_spec.rb#L39-L41). I think it's to avoid breaking apps that accidentally don't pass everything that's expected to a component.

Reply:
Hmmm...I'm not sure that one's a great precedent. I guess you mean *silently* breaking the app. That still seems the fault of the app. I'm content for it to stay in 👍.